references/plugins/references.go

```diff
@@ -527,8 +572,13 @@ func (ref *MarkdownReference) prepareTerraformOutputs(tableName string, paramete
 		return ""
 	}
 	refContent := fmt.Sprintf("\n\n%s\n\n", tableName)
-	refContent += "Name | Description\n"
-	refContent += "------------ | ------------- \n"
+	switch ref.I18N {
+	case Zh:
+		refContent += " 名字 | 描述 \n"
+	case En, "":
+		refContent += " Name | Description \n"
+	}
```

Comment:
please fine a folder for i18n descpritions.

Reply:
Done. Great suggestion.  Not only can all the words and phrases be used by other callers, but also does code get clean.